README.md

```diff
@@ -40,10 +40,31 @@ effectively update the UI to match the most recent tree with the minimum number
 
 > If you want to learn more about this, the [React documentation](https://reactjs.org/docs/reconciliation.html) contains an article that explains the choices made in React's diffing algorithm.
 
-# <a name="run-the-code"></a> :running: Run the code
+# <a name="test-your-impl"></a> :running: Testing your implementation
+
+We have provided you with tests for each task. We urge you to use these and run them after each task to verify you implementation. Additionally, you can run them with an incomplete implementation point you in the right direction.
+
+To run the tests for a specific task, you can simply specify the task (in this case task 1):
+
+```
+npm install
```

Comment:
Trenger vi å si `install` i alle kodesnuttene? Er det ikke like greit å bare ha en seksjon øverst som sier `first of all, npm install`...?